### PR TITLE
clean up legacy rpc

### PR DESCRIPTION
**Pull request by paulwe**

### CODE REVIEW

**Review by davidzhao — APPROVED** [review]
I can't tell you how happy I am to see this getting cleaned up! ❤️  Thank you Paul!

We'll still need to update config.go to remove [SignalRelay](https://github.com/livekit/livekit/blob/master/pkg/config/config.go#L75) and from config-sample.yaml

File: pkg/routing/redisrouter.go [hunk 1]
```diff
@@ -237,82 +173,9 @@ func (r *RedisRouter) WriteNodeRTC(_ context.Context, rtcNodeID string, msg *liv
 	return r.writeRTCMessage(rtcSink, msg)
 }
 
-func (r *RedisRouter) startParticipantRTC(ss *livekit.StartSession, participantKey livekit.ParticipantKey, participantKeyB62 livekit.ParticipantKey) error {
-	prometheus.IncrementParticipantRtcInit(1)
```
Comment (boks1971) [inline pkg/routing/redisrouter.go:241]: These are used to track join success rate IIRC. Is there a new place where this happens? Or we need to clean those stats up. I think those stats have not been tricky to track.

**Review by boks1971 — APPROVED** [review]
Very nice!!!

File: pkg/routing/redisrouter.go (hunk 1 above)
Comment (paulwe) [reply pkg/routing/redisrouter.go:241]: this was the legacy path. this happens in pkg/service/signal.go now

**Pull request merged**